docker-compose/owgw.env

```diff
@@ -67,3 +67,4 @@ KAFKA_BROKERLIST=kafka:9092
 #STORAGE_TYPE_MYSQL_PASSWORD=owgw
 #STORAGE_TYPE_MYSQL_DATABASE=owgw
 #STORAGE_TYPE_MYSQL_PORT=3306
+#CERTIFICATES_ALLOWMISMATCH=true
```

Comment:
As I mentioned in my other comment I think we should disable this by default.

Reply:
Yes, we've discussed this with Jaspreet in related task and decided to make it disabled by default